diagnostic.md

```diff
@@ -27,6 +31,7 @@ Record your responses inside the fenced code blocks below each question.
 
     ```html
     <!-- your response here -->
+    {{contacts/my-contact}}
```

Comment:
you don't need the `contacts/`.  So it will look like `my-contact` or:
```
{{#each model as |contact|}}
{{my-contact contact=contact}}
{{/each}}
```